### PR TITLE
[DEV-190648] Close custom tabs by redirect activity

### PR DESCRIPTION
**Pull request by andresguedes**
A new Activity was added to call the main screen, after **urlScheme** was called. In this time, the **Chrome Custom Tabs** is open and the user needed to close it manually. Now, this will close automatically after the **urlScheme** trigger.

### CODE REVIEW

**Review by marcosrivereto — APPROVED** [review]
👍

**Review by pedroabdenor — APPROVED** [review]
👍

**Review by lukevance — CHANGES_REQUESTED** [review]
Everything worked as expected! I just have one question, do these changes need to be made manually in the Android app? If so, we should add notes to the primary readme page so people understand which modifications must be made.

File: README.md [hunk 1]
```diff
@@ -75,3 +75,32 @@ handleOauthMessage = (message: any) => {
   }
 }
 ```
+
+## Closing Chrome Custom Tab in Android
```
Comment (lukevance) [inline README.md:79]: Perfect, thank you!

**Review by lukevance — APPROVED** [review]
Looks good!

**Pull request merged**